genx/setup.cfg

```diff
@@ -1,14 +1,51 @@
-[sdist]
-formats=gztar
+[metadata]
+name = GenX
+version = attr: genx.version.__version__
+author = Matts Bjorck, Artur Glavic
+author_email = artur.glavic@gmail.com
+description = X-ray and Neutron reflectivity fitting software.
+long_description = file: README.txt
+url = http://genx.sourceforge.net/
+license = GPL v3
+license_file = LICENSE.txt
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Operating System :: OS Independent
+    Topic :: Scientific/Engineering :: Physics
+    Development Status :: 6 - Mature
 
-[bdist]
-formats=rpm
```

Comment:
Is there any reason not to use rpm as default bdist? Mac and Windows will use pyinstaller, anyway.

Reply:
It just felt counterintuitive to me, as you can always use setup.py bdist_rpm to generate the rpm (as in make_deb.sh). If you liked it, I can revert it.